### DISCUSSION TITLE
Repository filtering on the PR search view is not working

### DISCUSSION THREAD

**Discussion by max-marcus** [community/community#67024] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.130.0 (109513371)
Device: iPhone 12 Pro (iOS 16.6.1)
</details>

**Comment by iskae** [comment]
Hi @max-marcus 👋 

Thanks for reaching out ❤️ 

We have been trying to reproduce the issue, but unfortunately not able to do so 😅 

Can you please provide some extra details on this? Are you trying to filter for a repository from `Home -> Pull Requests`? What steps did you take when filtering?

Thanks in advance 🙇

**Reply by Mhtpyurt** [reply]
Merhaba

**Comment by max-marcus** [comment]
Hi Aydin,Yeah I went from Home -> Pull RequestsThen I put on an Organization filter which worksThen I put on Repository filter which doesn’t work (still shows PRs from all org repositories)Made a screen capture video just now if that helpsOn Sep 15, 2023, at 7:21 AM, Aydin Emre Iskender ***@***.***> wrote:﻿
Hi @max-marcus 👋
Thanks for reaching out ❤️
We have been trying to reproduce the issue, but unfortunately not able to do so 😅
Can you please provide some extra details on this? Are you trying to filter for a repository from Home -> Pull Requests? What steps did you take when filtering?
Thanks in advance 🙇
